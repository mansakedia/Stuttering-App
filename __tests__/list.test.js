import React from 'react';
import renderer from 'react-test-renderer';

import SocialScreen from '../Demo/SocialScreen';

test('Renders coorectly', () => {
    const tree = renderer.create('<SocialScreen />').toJSON();
    expect(tree).toMatchSnapshot();
});