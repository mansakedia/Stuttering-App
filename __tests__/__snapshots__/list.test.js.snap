// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders coorectly 1`] = `"<SocialScreen />"`;
